tests/ThrottlingMiddlewareTest.php

```diff
@@ -21,8 +22,41 @@
  */
 class ThrottlingMiddlewareTest extends AbstractPackageTestCase
 {
+    /**
+     * Middleware to test.
+     *
+     * @var ThrottlingMiddleware
+     */
+    protected $middleware;
```

Comment:
This property appears unused? Did you change your mind about implementation, then forget this bit?